domain/src/Item.js

```diff
@@ -21,20 +21,39 @@ function calcAps(item) {
 
 function calcDps(item) {
     let res = (((item.minDmg + item.maxDmg) / 2) * item.aps);
-    if (item.critChance && item.critDmg) {
+    if (item.critChance > 0 && item.critDmg > 0) {
         if (item.critChance > 100) item.critChance = 100;
-        return (res + (res * (item.critChance / 100) * (item.critDmg / 100)));
+        res += (res * (item.critChance / 100) * (item.critDmg / 100));
     }
-    return res;
+    return res.toFixed(2);
 }
 
 function calcScore(item) {
```

Comment:
My dentist has a goood drugs... I said `item.type`, but I meant `aps`.  Sure `item.type` store in db.
I didn't think that you speak about 'calcScore', because this has only one selected line in the frame with the piece of code
And I think 'calcScore' is deprecated. We have `totalDps` which estimate item